### ISSUE TITLE
Issue on page /content/01/05_jupyterlab.html

### ISSUE THREAD

**Issue by donbowen** [LeDataSciFi/ledatascifi-2021#2]
@tdinhjason 
1. Please check all links
2. Please check for grammar
3. Does it make sense
4. Is any part of the page broken?

**Comment by tdinhjason** [comment]
1. Links 
- 👍
2. Grammar
- I didn't detect anything heinous, all good here! 
3. Does it make sense?

- [ ] 1.5.1.5, in the tip code block maybe also just add on a sentence to say the submenu "Playback speed" is how you control video speed
- [ ] 1.5.4 Extensions, point out that it's under the puzzle icon. Most Lehigh students that aren't in CS programs or have had coding experience tend leave everything on default settings and don't tinker with any customization options
- [ ] 1.5.4, I had to install node.js in order to install the spell checker extension. MacOS users who also installed homebrew can install this in terminal via `brew install node`
- [ ] 1.5.4, "Dark mode" could be expanded into "Dark Mode (Settings -> JupyterLab Theme -> JupyterLab Dark)" I think dark mode is popular enough where people would want to enable this but might get lot in all the new menus

4. Anything broken?
- [ ] a bit of text is cut out here
![Screen Shot 2021-02-03 at 11 47 10 AM](https://user-images.githubusercontent.com/31483561/106780058-9da33f80-6615-11eb-9008-05d943d04c07.png)

5. other comments
- nice call out on dark mode, much easier on the eyes and new to me.
- JupyterLab seems much more user-friendly than classic Jupyter Lab!

**Comment by donbowen** [comment]
Thanks for the close look! Closing the issue but opening [issue 5](#5). 

- [x] 1.5.1.5, in the tip code block maybe also just add on a sentence to say the submenu "Playback speed" is how you control video speed
- [x] 1.5.4 Extensions, point out that it's under the puzzle icon. Most Lehigh students that aren't in CS programs or have had coding experience tend leave everything on default settings and don't tinker with any customization options
- [x] 1.5.4, I had to install node.js in order to install the spell checker extension. MacOS users who also installed homebrew can install this in terminal via brew install node
- [x] 1.5.4, "Dark mode" could be expanded into "Dark Mode (Settings -> JupyterLab Theme -> JupyterLab Dark)" I think dark mode is popular enough where people would want to enable this but might get lot in all the new menus
- [ ] a bit of text is cut out here
   - It's fine on my screen. The only solution is to modify margins on the page via the CSS option... I'm not doing it.
